packages/web/src/pages/oauth-login-page/OAuthLoginPage.tsx

```diff
@@ -263,104 +410,226 @@ export const OAuthLoginPage = () => {
     response_mode
   ])
 
-  const formOAuthResponse = async (account: User) => {
-    let email: string
-    if (!userEmail) {
+  useEffect(() => {
```

Comment:
would we want a "getDeveloperApp" method in developer-app api?

Reply:
thought about this but decided against since we don't need the result to be cached or anything! but can play around and see if it makes the code simpler